pycbc/types/timeseries.py

```diff
@@ -563,6 +563,31 @@ def psd(self, segment_duration, **kwds):
                            seg_stride=seg_stride,
                            **kwds)
 
+    def get_gate_indices(self, time, window):
```

Comment:
Is this meant to match up to the 'gate' method? Is there a risk that they could diverge from each other? 

This raises a couple questions. (1) Is this really tied to gating or is this somewhat more general and could be articulated as such? (2) if it really is this specific to gating, does it make sense to have it more closely tied to the actual gate application function?

Reply:
@ahnitz This replaces the same code in the `gate` method. See the line removals at lines 608-609 below. I moved these lines to their own method because we need to be able to get the gate indices in the GatedGaussian models. Having a stand alone method to do that serves as a single point of truth.

I think that answers your question? Or did you mean another `gate` function?